### PR TITLE
Photographer can apply to a post or cancel it (function)

### PR DESCRIPTION
**Pull request by trongdatdinh1**

### CODE REVIEW

**Comment by trongdatdinh1** [comment]
ready

File: app/views/posts/_apply.html.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<%= form_for current_photographer.requests.build, remote: true do |f| %>
```
Comment (DuongNT1512) [inline app/views/posts/_apply.html.erb]: nên build trong controller nhé

File: app/views/posts/_cancel.html.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<%= form_for current_photographer.requests.find_by(post_id: post.id), method: :delete, remote: true do |f| %>
```
Comment (DuongNT1512) [inline app/views/posts/_cancel.html.erb]: find trong controller

File: app/views/requests/create.js.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+$("#request_form_<%= @post.id %>").html("<%= escape_javascript(render partial: ('posts/cancel'), locals: {post: @post}) %>");
```
Comment (DuongNT1512) [inline app/views/requests/create.js.erb]: dunfg j render nhé nhé

**Pull request merged**